### PR TITLE
TUNIC: Add option groups, fix option descriptions

### PR DESCRIPTION
**Pull request by ScipioWright**
## What is this fixing or adding?
Adds an option group to section off the logic options (since those are more "advanced" options that we want to section off).
Also changes the descriptions to not look weird with the new webhost update.

## How was this tested?
Ran webhost

## If this makes graphical changes, please attach screenshots.
N/A

### CODE REVIEW

File: worlds/tunic/options.py [hunk 1]
```diff
@@ -1,28 +1,36 @@
...
     internal_name = "keys_behind_bosses"
     display_name = "Keys Behind Bosses"
 
 
 class AbilityShuffling(Toggle):
-    """Locks the usage of Prayer, Holy Cross*, and the Icebolt combo until the relevant pages of the manual have been found.
+    """
+    Locks the usage of Prayer, Holy Cross*, and the Icebolt combo until the relevant pages of the manual have been found.
     If playing Hexagon Quest, abilities are instead randomly unlocked after obtaining 25%, 50%, and 75% of the required Hexagon goal amount.
     *Certain Holy Cross usages are still allowed, such as the free bomb codes, the seeking spell, and other player-facing codes.
```
Comment (Zunawe) [inline worlds/tunic/options.py:35]: If you're adding spaces between asterisks and text, there's another one here.

**Review by Zunawe — COMMENTED** [review]
It looks like you missed `LaurelsLocation`

File: worlds/tunic/options.py (hunk 1 above)
Comment (ScipioWright) [reply worlds/tunic/options.py:35]: Good catch, thanks!

**Review by Zunawe — APPROVED** [review]
Ran the webhost, hovered the tooltips, unfolded the option group

**Review by Exempt-Medic — APPROVED** [review]
Changes LGTM and display well when running WebHost

**Review by Ixrec — APPROVED** [review]
The one group added here looks good. From re-skimming all the other options, I think "Hexagon Quest" might make a decent group since that's 3 options, 2 of which mean nothing if you don't turn on the 1st one. But none of the others seem groupable enough for me to suggest anything else.

**Review by LegendaryLinux — APPROVED** [review]

**Pull request merged**